src/ui/dialogs/settings.ts

```diff
@@ -79,7 +79,7 @@ class ClientSection extends Section {
 
 class AccountOverviewSection extends Section {
    constructor(navigation: Navigation) {
-      super(navigation, Translation.t('Accounts'));
+      super(navigation, false, Translation.t('Accounts'));
```

Comment:
The title is a lot more common, so adding the new parameter `collapsible` to the end would simplify a lot.

Reply:
done